deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -349,6 +357,7 @@ extension NewChatViewController {
         )
         alert.addAction(UIAlertAction(title: String.localized("delete"), style: .destructive, handler: { [weak self] _ in
             self?.deleteContact(contactId: contactId, indexPath: indexPath)
+            callback?()
```

Comment:
callback is not called on all paths which means the UIContextualAction's completionHandler won't be called when you cancel.

Reply:
is it really needed that the completion handler is always called. 

how does eg. WhatsApp keep a swipe-action open cancelling a subsequent dialog (eg. at "more" there)

Reply:
AH so it is intentional, mb

Reply:
still, the name `callback()` is not-so-great, maybe better `confirmed()`

Reply:
Ah yeah or `didDelete()`